### ISSUE TITLE
Figure out how to distribute this tool as a Go module

### ISSUE THREAD

**Issue by kanishkdudeja** [kanishkdudeja/nginx-logs-replay#14]
We need to figure out a way to distribute this tool so that other projects can also use it as a dependency.

Since Vgo (versioned Go) is the official dependency management tool for Go now, we need to see how we can make this library compatible with it.

Most information about Vgo can be found here: https://github.com/golang/go/wiki/vgo

**Comment by mgill25** [comment]
Curious - why do we need this as a module? I consider this tool as a command line utility - something like a grep or curl or wget. Generally, a module or a library is something that can be imported and used inside other Go projects, but I don't think log replay has that kind of use case.

**Comment by kanishkdudeja** [comment]
> Curious - why do we need this as a module? I consider this tool as a command line utility - something like a grep or curl or wget. Generally, a module or a library is something that can be imported and used inside other Go projects, but I don't think log replay has that kind of use case.

Yes, I agree. I'll think more about potential use cases and get back to you.

In any case, I just logged this since this thought came to my mind. Adding the `Low priority` label to this issue.
